src/floki/core.cljs

```diff
@@ -31,22 +32,28 @@
 
 (defn convert
   [x]
-  (let [res (try
+  (let [tr (transit/reader :json)
```

Comment:
Im not proud of this try/catch thing :(
Any idea on how to improve this?

Reply:
I mean.. it gets the job done :) Promises may help flatten it, and be a bit kinder to the event loop if there is more going on concurrently. Guess in this case would have to consider what other requirements there are. Reduce would work too!